### ISSUE TITLE
false positive for Style/Mixin

### ISSUE THREAD

**Issue by marcandre** [bbatsov/rubocop#5261]
I get a complaint for the following code:

```
module X
  def foo
    include Bar
  end
end
```

"Style/MixinUsage: include is used at the top level. Use inside class or module"

No, it's *not* used at the top level.

```
$ rubocop -v
0.52.0
```

**Comment by pocke** [comment]
Definitely, the `include` is an instance method of `X`, so the cop should not complain about the code. cc: @koic

**Comment by koic** [comment]
Thanks for the feedback.  I confirmed this reproduction. I'm thinking about opening a PR for this issue later.
